README.md

```diff
@@ -180,6 +181,17 @@ Vue.config.errorHandler = (error) => {
 }
 ```
 
+## According to different pages or modules, add custom tags to spans.
+
+```js
+app.on('routeChange', function () {
+  ClientMonitor.setCustomTags([
```

Comment:
Could this method be called repeated to override tags list?

Reply:
> Could this method be called repeated to override tags list?

Yes, it can override custom tags.